draft-packages/tag/KaizenDraft/Tag/Tag.scss

```diff
@@ -227,6 +227,7 @@ $small: $spacing-md;
 $pulse-size-initial: 2px;
 $pulse-scale-factor: 9;
 $pulse-size-after: $pulse-size-initial * $pulse-scale-factor;
+$pulse-ring-position: calc(-1 * (#{$pulse-size-after} / 2) + (#{$pulse-size-initial} / 2));
```

Comment:
```suggestion
$pulse-ring-position: calc(-1 * calc(#{$pulse-size-after} / 2) + calc(#{$pulse-size-initial} / 2));
```
I think?

Reply:
I think because it's wrapped in a calc initially it doesn't need it as long as its wrapped in a parenthasis, just testing the theory now

Reply:
yep no warnings, console warning, so as long as its wrapped in an initial calc the parenthesis should suffice (and is a little more readable imo)